lib/factory_girl/registry.rb

```diff
@@ -13,6 +13,10 @@ def clear
       @items.clear
     end
 
+    def remove(name)
+      @items.delete(name.to_sym)
```

Comment:
I think it should _not_ raise.  If you try to delete a factory that was never there in the first place, the outcome is the same either way -- the factory is no longer there.  If it were to raise if not there, then you'd need a way to _check_ if it was there first, but i don't think it's neccesary, it should happily no-op if not there. I forget if there was a test for that, I guess there should be.